gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/GenerateDataTemplateTask.java

```diff
@@ -48,6 +48,7 @@
   private boolean _enableArgFile;
   private Boolean _generateLowercasePath;
   private boolean _generateFieldMask;
+  private Boolean _generateImported;
```

Comment:
What's the use of this new boolean value? I don't see it has been used anywhere.

Reply:
Oh, I just saw it.